content/docs/command-reference/update.md

```diff
@@ -42,34 +42,31 @@ $ dvc update --rev master
 
 ### Worktree update
 
-When using a
-[worktree](/doc/user-guide/data-management/cloud-versioning#worktree-remotes)
-remote, `dvc update` will update the specified target to match the current
-version of the corresponding file or directory from the remote storage. If the
-current version of the specified target is a deleted file or an empty directory,
-`dvc update` will fail.
+When using a cloud-versioned [worktree remote], this command will attempt to
+update its `targets` to match the current version on remote storage.
+
+[worktree remote]:
+  /doc/user-guide/data-management/cloud-versioning#worktree-remotes
 
 <admon type="warn">
 
-Note that the `--rev`, `--no-download` and `--to-remote` flags are not
-compatible when updating from a worktree remote.
+The `--rev`, `--no-download` and `--to-remote` options are not compatible.
 
 </admon>
 
 ## Options
 
-- `--rev <commit>` - commit hash, branch or tag name, etc. (any
-  [Git revision](https://git-scm.com/docs/revisions)) of the repository to
-  update the file or directory from. The latest commit in `master` (tip of the
-  default branch) is used by default when this option is not specified.
+- `--rev <version>` - commit hash, branch or tag name, etc. (any [Git revision])
+  of the repository to update the file or directory from. The latest commit in
+  `master` (tip of the default branch) is used by default.
```

Comment:
```suggestion
  `main` (tip of the default branch) is used by default.
```
💅

Reply:
Good catch. I'll make a separate 💅🏼 PR for this because there are more instances throughout docs...

Reply:
Wait. I think the default branch name [for Git in general](https://git-scm.com/book/en/v2/Git-Branching-Branches-in-a-Nutshell#:~:text=The%20default%20branch%20name%20in,the%20last%20commit%20you%20made.) is `master` so maybe keep as-is?

Reply:
I think all of GH, GL, and BB encourage `main`, and it's generally considered better practice at this point, right? Let's open a separate issue for it?